lib/manifest.js

```diff
@@ -31,7 +31,7 @@ export async function generateManifest({
   manifestPath = 'build/manifest.json',
 } = {}) {
   const manifestDir = path.dirname(manifestPath);
-  const files = glob.sync(pattern);
+  const files = globSync(pattern).sort();
```

Comment:
On version 8, glob.sync returned results ordered, which is not the case with version 9's globSync. Adding the .sort() to make sure we keep the same behavior.